packages/net/http/_client_connection.pony

```diff
@@ -181,6 +181,14 @@ actor _ClientConnection is HTTPSession
     """
     None
 
+  be dispose() =>
+    """
+    Close the connection from the client end.
+    """
+    match _conn
+      | let c: TCPConnection => c.dispose()
```

Comment:
the standard formatting in the library is for the | to be under the 'm' in match.

we need to write up a style guide. sorry for that.